lib/flipper/railtie.rb

```diff
@@ -0,0 +1,21 @@
+module Flipper
+  class Railtie < Rails::Railtie
+    initializer "flipper.memoizer" do |app|
+      config = Flipper.configuration
+
+      if config.memoize
+        app.middleware.use Flipper::Middleware::Memoizer, {
+          env_key: config.env_key,
+          preload: config.preload,
+          if: config.memoize.is_a?(Proc) ? config.memoize : nil
```

Comment:
Should we just check that it responds to `call`? That would allow:

```ruby
class Blah
  def self.call(request)
    # true or false
  end
end

Flipper.configure do |config|
  config.memoize = Blah
end
```

Reply:
Great idea. Fixed!